### DISCUSSION TITLE
[交流灌水]今日份的反米魔怔人乐子

### DISCUSSION THREAD

**Discussion by youyiMC** [share121/inter-knot#2979] [其他]
省流：一群妄想攻击GitHub的所谓小学生黑客<img src="https://img-static.mihoyo.com/communityweb/upload/460474087917ad69a4ff975a3132a0df.png" alt="妮可 哈哈" width="75" >

<iframe src="//player.bilibili.com/player.html?isOutside=true&aid=112952103406877&bvid=BV137Yze6EZv&cid=500001647587605&p=1" scrolling="no" border="0" frameborder="no" framespacing="0" allowfullscreen="true"></iframe>

**Comment by youyiMC** [comment]
啊嘞？好像没有成功插入b站视频<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/7cb87b286c3d062300d31224508c0c9e_8768116406867668880.png" alt="丹恒 惊" width="75" >

**Comment by cpu644** [comment]
你插入失败了。直接贴链接吧
